enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -237,7 +237,6 @@ if (hasPremiumFeature("whitelabel")) {
 
   PLUGIN_APP_INIT_FUCTIONS.push(() => {
     updateColors();
-    MetabaseSettings.on("application-colors", updateColors);
```

Comment:
This is the part where we change the color without requiring reloading the page. Please see https://www.loom.com/share/c64729d5b60847388f1ba0bd2efcc4c2?sid=3f3a0ac0-78ba-443c-ac5d-cd8f5000a8fd at minute 4:20 for context.